AdminApi/Controllers/MetricsController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using AdminApi.Mvc;
+using Backbone.Modules.Quotas.Application.DTOs;
+using Backbone.Modules.Quotas.Application.Metrics.Queries.ListMetrics;
+using Enmeshed.BuildingBlocks.API;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AdminApi.Controllers;
+
+[Route("api/v1/[controller]")]
+
+public class MetricsController : ApiControllerBase
+{
+    public MetricsController(IMediator mediator) : base(mediator) { }
+
+    [HttpGet]
+    [ProducesResponseType(typeof(HttpResponseEnvelopeResult<MetricDTO>), StatusCodes.Status200OK)]
```

Comment:
Try to concentrate a little bit.

```suggestion
    [ProducesResponseType(typeof(HttpResponseEnvelopeResult<ListClientsResponse>), StatusCodes.Status200OK)]
```